girder/models/model_base.py

```diff
@@ -1261,11 +1261,12 @@ def copyAccessPolicies(self, src, dest, save=False):
         return dest
 
     def filterResultsByPermission(self, cursor, user, level, limit=0, offset=0,
-                                  removeKeys=()):
+                                  removeKeys=(), flags=None):
         """
         Given a database result cursor, this generator will yield only the
         results that the user has the given level of access on, respecting the
-        limit and offset specified.
+        limit and offset specified. It can also only yield results for which the
```

Comment:
Users can only see resources to which they have at least READ access, so we always filter by level.

I'll change the docstring to be clearer.